web/docs/data-model/crud.md

```diff
@@ -220,13 +224,17 @@ export const createTask: Tasks.CreateAction<CreateTaskInput, Task> = async (
 }
 ```
 
+Wasp automatically generates the type `Tasks.CreateAction` type namespace based on the CRUD declaration in your Wasp file.
```

Comment:
> type `Tasks.CreateAction` type namespace

Feels off, maybe

```suggestion
Wasp automatically generates the `Tasks.CreateAction` type based on the CRUD declaration in your Wasp file.
```

Reply:
Yeah, that's a mistake. Leftover from the previous sentence.